### PR TITLE
Add Greg and Sebastian as Editors to VC Data Integrity

### PR DESCRIPTION
**Pull request by msporny**
This PR adds Greg and Sebastian as Editors to the Data Integrity specification. The PR shouldn't be processed until after Sebastian's IE application has been processed. /cc @Wind4Greg @seabass-labrax

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-integrity/pull/113.html" title="Last updated on Jul 13, 2023, 9:52 PM UTC (60342e4)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-integrity/113/211352d...60342e4.html" title="Last updated on Jul 13, 2023, 9:52 PM UTC (60342e4)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -50,6 +50,12 @@
             name: "Dave Longley", url: "https://digitalbazaar.com/",
             company: "Digital Bazaar", companyURL: "http://digitalbazaar.com/",
             note: "2014-2022", w3cid: 48025
+          }, {
+            name: "Greg Bernstein", url: "https://www.grotto-networking.com/",
+            company: "Invited Expert", w3cid: 0
```
Comment (msporny) [inline index.html]: @Wind4Greg, I need you to fill out your w3cid here. You can find out what it is by going here: 

https://www.w3.org/users/myprofile/

The URL will change into something that looks like this:

https://www.w3.org/users/41758/

Whatever number shows up in the URL is your w3cid.

File: index.html [hunk 2]
```diff
@@ -50,6 +50,12 @@
...
+          }, {
+            name: "Sebastian Crane", url: "https://github.com/seabass-labrax",
+            company: "Invited Expert", w3cid: 0
```
Comment (msporny) [inline index.html]: @seabass-labrax, I need you to fill out your w3cid here. You can find out what it is by going here: 

https://www.w3.org/users/myprofile/

The URL will change into something that looks like this:

https://www.w3.org/users/41758/

Whatever number shows up in the URL is your w3cid.

**Review by dlongley — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (Wind4Greg) [inline index.html]:
```suggestion
            company: "Invited Expert", w3cid: 140479
```

**Review by Wind4Greg — APPROVED** [review]

**Comment by iherman** [comment]
As mentioned [elsewhere](https://github.com/w3c/vc-di-eddsa/pull/50#issuecomment-1628305781) we should receive a formal IE application from @seabass-labrax. As said in the PR description, this PR _MUST NOT_ be merged until that is settled.

File: index.html (hunk 2 above)
Comment (seabass-labrax) [inline index.html]:
```suggestion
            company: "Invited Expert", w3cid: 140132
```

**Review by seabass-labrax — CHANGES_REQUESTED** [review]

**Comment by msporny** [comment]
Editorial, multiple reviews, @seabass-labrax and @Wind4Greg are now WG members, no objections, merging.

**Pull request merged**